system/classes/Kohana/Core.php

```diff
@@ -233,7 +233,7 @@ public static function init(array $settings = NULL)
 					// Set permissions (must be manually set to fix umask issues)
 					chmod($settings['cache_dir'], 0755);
 				}
-				catch (Exception $e)
+				catch (Throwable $e)
```

Comment:
Why do you use Throwable instead of Exception?

Reply:
Need catch Error (Error/Exception implements Throwable).